.github/dependabot.yml

```diff
@@ -52,82 +52,82 @@ updates:
     interval: daily
 
 - package-ecosystem: "docker"
-  directory: "/examples/dynamic-config-cp"
+  directory: "/examples/ext_authz"
   schedule:
     interval: daily
 
 - package-ecosystem: "docker"
-  directory: "/examples/ext_authz"
+  directory: "/examples/fault-injection"
   schedule:
     interval: daily
 
 - package-ecosystem: "docker"
-  directory: "/examples/ext_authz/auth/http-service"
+  directory: "/examples/grpc-bridge"
   schedule:
     interval: daily
 
 - package-ecosystem: "docker"
-  directory: "/examples/ext_authz/auth/grpc-service"
+  directory: "/examples/kafka"
   schedule:
     interval: daily
 
 - package-ecosystem: "docker"
-  directory: "/examples/fault-injection"
+  directory: "/examples/local_ratelimit"
   schedule:
     interval: daily
 
 - package-ecosystem: "docker"
-  directory: "/examples/grpc-bridge"
+  directory: "/examples/mysql"
   schedule:
     interval: daily
 
 - package-ecosystem: "docker"
-  directory: "/examples/grpc-bridge/client"
+  directory: "/examples/opentelemetry"
   schedule:
     interval: daily
 
 - package-ecosystem: "docker"
-  directory: "/examples/grpc-bridge/server"
+  directory: "/examples/redis"
   schedule:
     interval: daily
 
 - package-ecosystem: "docker"
-  directory: "/examples/load-reporting-service"
+  directory: "/examples/shared/build"
   schedule:
     interval: daily
 
 - package-ecosystem: "docker"
-  directory: "/examples/local_ratelimit"
+  directory: "/examples/shared/echo"
   schedule:
     interval: daily
 
 - package-ecosystem: "docker"
-  directory: "/examples/lua"
```

Comment:
ditto?

Reply:
i think `lua` was there for `echo` Dockerfile